pkg/controller/daemonset/daemonset_util.go

```diff
@@ -163,15 +164,30 @@ func NodeShouldRunDaemonPod(client client.Client, node *corev1.Node, ds *appsv1a
 	return
 }
 
+func getSchedulerNodeInfo(node *corev1.Node) *schedulernodeinfo.NodeInfo {
```

Comment:
How about rename to `newSchedulerNodeInfo`?

Reply:
ok